Classes/Console/Database/Process/MysqlCommand.php

```diff
@@ -141,7 +158,7 @@ private function createTemporaryMysqlConfigurationFile()
             $passwordDefinition = sprintf('password="%s"', addcslashes($this->dbConfig['password'], '"\\'));
         }
         $confFileContent = <<<EOF
-[mysqldump]
+[' . $binary . ']
```

Comment:
```suggestion
[client]
```

AFAIS the `client` group is supported by all CLI clients:

- `mysql`: https://dev.mysql.com/doc/refman/8.4/en/option-file-options.html
- `mysqldump`: https://dev.mysql.com/doc/refman/8.4/en/mysqldump.html#mysqldump-option-file-options
- `mariadb`: https://mariadb.com/kb/en/mariadb-command-line-client/#option-groups
- `mariadb-dump`: https://mariadb.com/kb/en/mariadb-dump/#option-groups